tutorials/forward/80_fix_bem_in_blender.py

```diff
@@ -207,6 +207,27 @@
 # mne.write_head_bem(op.join(head_dir, 'sample-head.fif'), coords, faces,
 #                    overwrite=True)
 
+# Blender editing tips
```

Comment:
Did not render properly

https://37211-1301584-gh.circle-artifacts.com/0/dev/auto_tutorials/forward/80_fix_bem_in_blender.html#sphx-glr-auto-tutorials-forward-80-fix-bem-in-blender-py

```suggestion
# %%
# Blender editing tips
```